analyzers/tests/SonarAnalyzer.Test/TestCases/SecureRandomSeedsShouldNotBePredictable.cs

```diff
@@ -254,6 +256,14 @@ void SecureRandom_Arrays(byte b, char[] whoKnows)
         sr.SetSeed(Convert.FromBase64CharArray(whoKnows, 0, 42));
         sr.Next(); // Compliant
 
+        sr = SecureRandom.GetInstance("SHA256PRNG", false);
+        sr.Next(); // Noncompliant
+        var ss = new byte[3];
+        ss[0] = b;
+        ss.Initialize();
```

Comment:
This sets each element of the array to the default value, so I consider it predictable.